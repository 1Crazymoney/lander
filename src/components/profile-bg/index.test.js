import React from 'react';
import ProfileBg from './index';
import renderer from 'react-test-renderer';

import wrapWithIntl from '../../utils/test-helper';

it('render component', () => {

  const props = {
    bg: {
      image: 'wave.jpg',
      color: '#cccccc',
      blur: 3
    }
  };

  const component = wrapWithIntl(<ProfileBg {...props} />);

  const tree = renderer
    .create(component)
    .toJSON();

  expect(tree).toMatchSnapshot();
});