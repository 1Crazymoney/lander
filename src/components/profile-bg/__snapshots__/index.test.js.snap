// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component 1`] = `
<div
  className="bg"
  style={
    Object {
      "backgroundColor": "#cccccc",
      "backgroundImage": "url('wave.jpg')",
      "filter": "blur(3px)",
      "height": "calc(100% + 12px)",
      "left": "-6px",
      "top": "-6px",
      "width": "calc(100% + 12px)",
    }
  }
/>
`;
